phpunit.xml

```diff
@@ -11,6 +11,10 @@
         <env name="SHELL_VERBOSITY" value="-1" />
     </php>
 
+    <listeners>
```

Comment:
I think PHPunit parses the .dist file if you don't  have a local version, right? (so there is no need to rename and add to the repo, you can just change this in the .dist and leave the option to override if some wants to test things differently on a local environment)

Reply:
:+1: I'll rename it (back) to `phpunit.xml.dist`!